packages/react/src/forms/ControlGroup/ControlGroup.module.css

```diff
@@ -0,0 +1,65 @@
+.ControlGroup__container {
+  display: flex;
+  flex-direction: column;
+  gap: var(--base-size-8);
+  font-family: var(--brand-body-fontFamily);
+  font-size: var(--brand-text-size-100);
+  border: none;
+  margin: 0;
+  padding: 0;
+}
+
+.ControlGroup__label {
+  font-weight: var(--base-text-weight-semibold);
+  margin: 0;
+  padding: 0;
+  margin-bottom: var(--base-size-8);
+}
+
+.ControlGroup__caption {
+  font-size: var(--brand-text-size-100);
+  letter-spacing: var(--brand-text-letterSpacing-100);
+  line-height: var(--brand-text-lineHeight-100);
+  color: var(--brand-color-text-muted);
+}
+
+.ControlGroup__validation {
+  display: flex;
+  gap: var(--base-size-4);
+  font-size: var(--brand-text-size-100);
+  font-weight: var(--base-text-weight-regular);
+  line-height: var(--brand-text-lineHeight-100);
+}
+
+.ControlGroup__validation--success {
+  color: var(--brand-color-success-fg);
+}
+
+.ControlGroup__validation--error {
+  color: var(--brand-color-error-fg);
+}
+
+.ControlGroup__validation-icon--success {
+  position: relative;
+  top: -1.6px;
```

Comment:
Just curious why this is -1.6px vs -1px? I think -1px looks more vertically aligned to the center, but this rule is so specific I'm probably missing something:

With -1px:

<img width="377" alt="Screenshot 2024-11-27 at 13 47 43" src="https://github.com/user-attachments/assets/c8b2abb6-7807-4c01-8bb4-f04c176e69b0">

Reply:
Again, pulled from FormControl to keep things consistent

https://github.com/primer/brand/blob/main/packages/react/src/forms/FormControl/FormControl.module.css#L94-L97

Happy to modify it if you'd prefer though?

Reply:
Ah, we've had this conversation before, just in the other direction 😄

It was 1px, but after a brief conversation ([here](https://github.com/primer/brand/pull/668#issuecomment-2250980453)) we settled on 1.6px.

https://github.com/primer/brand/pull/668/commits/4f50c91eea0331d01fe9157e81d1d6bd8434516d#diff-66f88db62ba93924cf6de6ae216861c98149f0122e02c93173b5778c43cd4b57R95-R98

Reply:
Ah okay 🤣 .. let's keep it the same then.. i used a visual guide in the other screenshot so that's probably more correct. Although could this be a bit different because the text is 100 vs 200?